src/main/java/dev/ebrydeu/spring_boot_library/controllers/MessageController.java

```diff
@@ -0,0 +1,46 @@
+package dev.ebrydeu.spring_boot_library.controllers;
+
+import dev.ebrydeu.spring_boot_library.domain.dto.MessageDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/Messages")
+@RequiredArgsConstructor
+public class MessageController{
+    private final MessageService messageService;
+
+    @GetMapping
+    public List<MessageDto> getAllMessages(){
+        return messageService.findAllMessages();
+    }
+
+    @GetMapping("/author/{author}")
+    public List<MessageDto> getMessagesByAuthor(@PathVariable String author){
+        return messageService.findMessagesByAuthor(author);
+    }
```

Comment:
#20  Now, That I look Att it what exactly String author supposed to be?  Is it Id name or just POJO? Does not make sense for me as for now.

Reply:
Should we consider just skipping author all together, or is the one who created the message also the author?

Reply:
You can skip for now. We will se how it will look without it

Reply:
I think you are right, my understanding the author is the one who created message, meaning we should be able to access this from UserEntity by id, and therefore do not need this in MessageEntity